turbolinks/src/main/java/com/basecamp/turbolinks/TurbolinksSession.java

```diff
@@ -769,4 +795,9 @@ private void validateRequiredParams() {
             throw new IllegalArgumentException("TurbolinksSession.visit(location) location value must not be null.");
         }
     }
+
+    @Override
+    public boolean canChildScrollUp() {
```

Comment:
I'd move this method up into either 1) the public block we have starting in line 592, or 2) maybe it's own new block that is for interface implementations?

We typically reserve the bottom block for private methods.